lv_sim/CMakeLists.txt

```diff
@@ -0,0 +1,146 @@
+file(GLOB_RECURSE SOURCES  "lv_drivers/*.c" "../src/libs/lvgl/src/*.c" )
+
+add_executable(main main.cpp mouse_cursor_icon.c ${SOURCES} ${INCLUDES})
+#message("add_executable(main main.c mouse_cursor_icon.c ${SOURCES} ${INCLUDES})")
+
+target_sources(main PUBLIC
+  sim/displayapp/DisplayApp.h
+  sim/displayapp/DisplayApp.cpp
+  sim/displayapp/LittleVgl.h
+  sim/displayapp/LittleVgl.cpp
+  sim/components/battery/BatteryController.h
+  sim/components/battery/BatteryController.cpp
+  sim/components/ble/MusicService.h
+  sim/components/ble/MusicService.cpp
+  sim/components/brightness/BrightnessController.h
+  sim/components/brightness/BrightnessController.cpp
+  sim/components/fs/FS.h
+  sim/components/fs/FS.cpp
+  sim/components/heartrate/HeartRateController.h
+  sim/components/heartrate/HeartRateController.cpp
+  sim/components/motion/MotionController.h
+  sim/components/motion/MotionController.cpp
+  sim/components/motor/MotorController.h
+  sim/components/motor/MotorController.cpp
+  sim/systemtask/SystemTask.h
+  sim/systemtask/SystemTask.cpp
+  # FreeRTOS
+  sim/FreeRTOS.h
+  sim/task.h
+  sim/tasks.cpp
+  # src/FreeRTOS
+  sim/portmacro_cmsis.h
+  # nrf
+  sim/libraries/log/nrf_log.h
+  # nrf/components/libraries/timer
+  sim/libraries/timer/app_timer.h
+  sim/libraries/timer/app_timer.cpp
+  )
+target_include_directories(main PRIVATE "sim")
+target_include_directories(main PRIVATE "sim/displayapp")
+target_include_directories(main PRIVATE "sim/displayapp/screens")
+target_include_directories(main PRIVATE "sim/libraries/timer") # for app_timer.h
+
+target_compile_definitions(main PRIVATE LV_CONF_INCLUDE_SIMPLE)
+target_sources(main PUBLIC
+  # LVGL Fonts
+  ../src/libs/lvgl/src/lv_font/lv_font_montserrat_14.c
+  ../src/libs/lvgl/src/lv_font/lv_font_montserrat_18.c
+  ../src/libs/lvgl/src/lv_font/lv_font_montserrat_22.c
+  ../src/libs/lvgl/src/lv_font/lv_font_montserrat_28.c
+
+  ../src/displayapp/fonts/lv_font_navi_80.c
+
+  ../src/displayapp/fonts/jetbrains_mono_extrabold_compressed.c
+  ../src/displayapp/fonts/jetbrains_mono_bold_20.c
+  ../src/displayapp/fonts/jetbrains_mono_76.c
+  ../src/displayapp/fonts/jetbrains_mono_42.c
+  ../src/displayapp/fonts/lv_font_sys_48.c
+  ../src/displayapp/fonts/open_sans_light.c
+)
+
+target_include_directories(main PRIVATE "${CMAKE_CURRENT_SOURCE_DIR}")
+target_include_directories(main PRIVATE "../src/libs")
+target_include_directories(main PRIVATE "lv_drivers")
+
+find_package(SDL2 REQUIRED SDL2)
+target_link_libraries(main PRIVATE SDL2::SDL2)
+
+
+# freeRTOS headers
+#target_include_directories(main PRIVATE "../src") # include for FreeRTOSConfig.h
+#target_include_directories(main PRIVATE "../src/FreeRTOS") # include for FreeRTOSConfig.h
+#target_sources(main PRIVATE
+#  ../src/FreeRTOS/port_cmsis_systick.c
+#  ../src/FreeRTOS/port_cmsis.c
+#  ../src/FreeRTOS/port.c)
+##target_include_directories(main PRIVATE "/home/nero/repos/pinetime/FreeRTOS-LTS/FreeRTOS/FreeRTOS-Kernel/include")
```

Comment:
A bunch of things like these have been commented out, if they're not required can we delete them?

There's also mentions of your home folder that should be removed.

Reply:
yes, will remove, thanks for looking into it!

Reply:
updated